org.aliflang.lang.yml

```diff
@@ -0,0 +1,149 @@
+      - g++ -o "bin/alifstudio" "alifstudio.o" -L"/app/include" -L"/app/usr/local/include/aliflibwx/" -L"/app/include/wx-3.1-alif/wx/" -L"/lib" -L"/app/lib"  -lwx_gtk2u_alif_webview-3.1 -lwx_gtk2u_alif_propgrid-3.1 -lwx_gtk2u_alif_aui-3.1 -lwebkitgtk-1.0 -lgtk-x11-2.0 -lgdk-x11-2.0 -lpangocairo-1.0 -latk-1.0 -lcairo -lgdk_pixbuf-2.0 -lpangoft2-1.0 -lpango-1.0 -lfontconfig -lfreetype -lsoup-2.4 -lgio-2.0 -lgobject-2.0 -ljavascriptcoregtk-1.0 -lglib-2.0 -lwx_gtk2u_alif_stc-3.1 -lwx_gtk2u_alif_core-3.1 -lwx_baseu_alif-3.1 -lwxscintilla_alif-3.1 -lwxtiff_alif-3.1 -lwxjpeg_alif-3.1 -lwxpng_alif-3.1 -lgtk-x11-2.0 -lgdk-x11-2.0 -lpangocairo-1.0 -latk-1.0 -lcairo -lgdk_pixbuf-2.0 -lgio-2.0 -lpangoft2-1.0 -lpango-1.0 -lgobject-2.0 -lfontconfig -lfreetype -lgthread-2.0 -pthread -lglib-2.0 -lX11 -lXxf86vm -lSM -lgtk-x11-2.0 -lgdk-x11-2.0 -lpangocairo-1.0 -latk-1.0 -lcairo -lgdk_pixbuf-2.0 -lgio-2.0 -lpangoft2-1.0 -lpango-1.0 -lgobject-2.0 -lglib-2.0 -lfontconfig -lfreetype -lpangoft2-1.0 -lpango-1.0 -lgobject-2.0 -lglib-2.0 -lfontconfig -lfreetype -lwxzlib_alif-3.1 -lwxregexu_alif-3.1 -lwxexpat_alif-3.1 -pthread -m64 -static-libgcc -static-libstdc++ -ldl -lm -ldl -lm -O2;
+      - install bin/alifstudio /app/usr/local/bin/alifstudio;
+      - ln -s /app/usr/local/bin/alifstudio /app/bin/alifstudio;
+    sources: 
+      - type: git
+        url: https://github.com/aliflanguage/alifstudio.git
+        branch: flatpak-fork-unstable
+        commit: 9e042b8dfe176065c2c7a2866ed105d7ee5ad6e3
+
+
+  - name: desktop-files
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/share/icons/hicolor/32x32/apps/ /app/share/metainfo/ /app/share/applications/
+      - install "org.aliflang.lang.png" /app/share/icons/hicolor/32x32/apps/org.aliflang.lang.png
+      - install "org.aliflang.lang.metainfo.xml" /app/share/metainfo/
+      - install "org.aliflang.lang.desktop" /app/share/applications/
+      - ln -s /app/share/applications /app/usr/share/applications
+      - ln -s /app/share/appdata /app/usr/share/appdata
+      - ln -s /app/share/icons /app/usr/share/icons
```

Comment:
```suggestion
```
you don't need those

Reply:
If I install icon only in `/app/share/icons/hicolor/128x128/apps/` i get this error 
`Renaming org.aliflang.lang.metainfo.xml to share/appdata/org.aliflang.lang.appdata.xmlRunning appstream-composeProcessing application org.aliflang.langError loading desktop file: Failed to find icon: Failed to find icon org.aliflang.langError: ERROR: appstream-compose failed: Child process exited with code 1`

If i install icon only in `/app/usr/share/icons/hicolor/128x128/apps/` , he can't export app icon
`WARNING: Icon referenced in desktop file but not exported: org.aliflang.lang`


**I fixed it when installing the icon in both path**